_includes/releases/babelfish-1.1.1.md

```diff
@@ -0,0 +1,121 @@
+
+- Babelfish Version: 1.1.1
+- PostgreSQL Server Version: 13.5
+- Download source distributions:
+  - [BABEL_1_1_1__PG_13_5.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_1_1__PG_13_5/BABEL_1_1_1__PG_13_5.zip)
+  - [BABEL_1_1_1__PG_13_5.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_1_1__PG_13_5/BABEL_1_1_1__PG_13_5.tar.gz)
+- Babelfish Compass
+  - [Download](https://github.com/babelfish-for-postgresql/babelfish_compass/releases)
+- Date: November 8, 2022
+
+## Overview
+
+Babelfish 1.1.1 contains all of the features and limitations of version 1.1.0:
+
+* Enhancements and fixes that improve the user experience, listed below.
+* Additional built-in functions, including `ORIGINAL_LOGIN()`, `SQUARE()` and more.  
+* Additional system views, including `sys.columns`, `sys.sysindexes`, and more.
+* Additional system stored procedures, including `sp_columns`, `sp_statistics`, and more.
```

Comment:
These features were for version 1.1.0, for 1.1.1 only fix are security enhancements.